charts/stable/plaxt/docs/installation_notes.md

```diff
@@ -6,7 +6,7 @@ title: Installation Notes
 
 - Please create an [app](https://trakt.tv/oauth/applications/new) in trakt.tv.
 - Set the trakt app name to whatever you like for ex: `plaxt`.
-- Optionally set an icon, you can download this [icon](/img/hotlink-ok/chart-icons/plaxt.png).
+- Optionally set an icon, you can download this [icon](./img/icon.webp).
```

Comment:
Use the full link to the hot link with wepb ext and remove the icoj under the docs folder